jestSetupFileAfterEnv.ts

```diff
@@ -0,0 +1,5 @@
+import { resetToDefaults } from './src/pure';
+
+beforeEach(() => {
```

Comment:
It is indeed not needed for every test file but we can't be sure that we'll call it in each test file, this is like a safety net that ensures our tests don't depend on each other. Also I don't see a reason why we wouldn't want to reset between each test so I think it's better if it's global

Reply:
Maybe it our case it makes sense, I've based my comment on that in a typical user app using RNTL this would not make sense as it would override any `configure` changes made in a global Jest setup file.